client/src/App.js

```diff
@@ -76,35 +76,14 @@ class App extends Component {
     this.handleSignup = handleSignup.bind(this);
     this.handleLogin = handleLogin.bind(this);
     this.handleLogout = handleLogout.bind(this);
+    this.getUserData = getUserData.bind(this);
   }
   componentDidMount() {
-    if (this.state.logged_in) {
-      fetch(`${BASE_URL}/auth/user`, {
-        headers: {
-          Authorization: `Token ${localStorage.getItem("token")}`
-        }
-      })
-        .then(res => res.json())
-        .then(json => {
-          // console.log(json);
-          if (json.detail == "Invalid token.") {
-            this.handleLogout();
-          } else {
-            this.setState(
-              {
-                email: json.email,
-                first_name: json.profile.first_name,
-                image: json.profile.image,
-                userid: json.id,
-                profileLoaded: true
-              },
-              this.logState
-            );
-          }
-        });
-    }
+    if (this.state.logged_in) this.getUserData()
```

Comment:
Can you format this properly, with brackets?

Reply:
done